### PR TITLE
Add NStepGRU link converter example to ONNX-Chainer test

### PR DESCRIPTION
**Pull request by msakai**
This depends on #8492 and #8493, and I made this PR as a draft PR because it contains a workaround to avoid `ValidationError`.

### CODE REVIEW

**Comment by msakai** [comment]
As #8492, #8493, and #8495 are already merged, I rebased the PR to simplify it and removed `Draft` label to start discussion.

The remaining problem is that it contains a workaround (that replace `Permutate` converter) to avoid `ValidationError`. The problem is that `context.get_name(func.indices)` returns correct name if `func.indices` is an array given in `args` of `onnx_chainer.export`, but otherwise it returns generated name (e.g. "v330") which is not output of any node.

One possible solution (other than the current workaround) is to call `add_const` if necessary (in `convert_Permutate` or in `Context.get_name`). But the down side is that it may hide disconnection of computation graph and silently produce unintended ONNX graph. And I have no idea what is the ideal fix.

Anyway, Chainer is now in maintenance mode, and if it is difficult to fix, it is totally fine giving up fixing the issue or giving up merging this PR.

**Comment by take-cheeze** [comment]
Jenkins, test this please!

File: tests/onnx_chainer_tests/functions_tests/test_rnn.py [hunk 1]
```diff
@@ -44,3 +47,64 @@ def __call__(self, hx, ws1, ws2, ws3, bs, xs):
...
+
+            def __call__(self, *xs):
+                hy, ys = self.gru(None, xs)
+                return [hy] + ys
+
+        model = Model()
+        xs = [input_generator.increasing(seq_length, input_size)
+              for i in range(batch_size)]
+
+        # XXX: Replace Permutate converter for avoiding error like:
```
Comment (disktnk) [inline tests/onnx_chainer_tests/functions_tests/test_rnn.py]: Please set as `NOTE(username):` ref: https://github.com/chainer/chainer/blob/b1a9f55015ef99f2ba4803d8b3cc2d0aa81ba736/onnx_chainer/functions/array.py#L519

Comment (msakai) [reply tests/onnx_chainer_tests/functions_tests/test_rnn.py]: Thanks. I replaced "XXX:" with "NOTE(msakai):" as suggested.

Comment (take-cheeze) [reply tests/onnx_chainer_tests/functions_tests/test_rnn.py]: Thank you!

**Comment by take-cheeze** [comment]
Jenkins test this please!

**Comment by take-cheeze** [comment]
Seems failure in https://ci.preferred.jp/chainer.py37.cpu/50117/ is fixed by: https://github.com/chainer/chainer/pull/8521

**Pull request merged**